src/styles/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  width: &$week-size * $day-size;
```

Comment:
check the syntax, cause now it's invalid
<img width="189" alt="Screenshot 2023-06-26 at 22 23 25" src="https://github.com/mate-academy/layout_calendar/assets/117587463/4daffdc6-9d5d-4046-988f-3339e01a6b43">